tests/rest-api/methods.yml

```diff
@@ -106,7 +106,12 @@
             "dependencies": [],
             "deployments": [],
             "disk": 0.0,
-            "env": {},
+            "env": {
+                "VAR": "VALUE",
+                "SECRET1": {
+                    "secret": "SECRET_VALUE"
```

Comment:
Are we actually using the new values in any test? It doesn't seem so. We should probably extend/add a test.